### PR TITLE
Migrate the script to use per-customer-id endpoint and auth by default

### PR DESCRIPTION
**Pull request by chenjiayuan**

### CODE REVIEW

**Comment by chenjiayuan** [comment]
DO NOT MERGE YET

**Review by arthur-observe — CHANGES_REQUESTED** [review]
Need to checkin with @jlebaugh and @mwiser-1 to see how this affects apps

File: README.md [hunk 1]
```diff
@@ -35,11 +35,11 @@ Run --help command for list of flags and options
 - Required --customer_id YOUR_OBSERVE_CUSTOMERID 
 - Required --ingest_token YOUR_OBSERVE_DATA_STREAM_TOKEN 
 ## Optional inputs
-- Optional --observe_host_name - Defaults to https://collect.observe.com/ 
-- Optional --config_files_clean TRUE or FALSE - Defaults to FALSE 
-    controls whether to delete created config_files temp directory
-- Optional --ec2metadata TRUE or FALSE - Defaults to FALSE 
-    controls fluentbit config for whether to use default ec2 metrics 
+- Optional --observe_host_name - Defaults to https://<YOUR_OBSERVE_CUSTOMERID>collect.observe.com/
```
[the remark this answers is not in the document]
Comment (chenjiayuan) [reply README.md:38]: Fixed.

**Review by mwiser-1 — COMMENTED** [review]
Works for me - no issues for host monitoring or linux host

**Review by jlebaugh — APPROVED** [review]

**Pull request merged**